tests/feature_extraction/test_feature_calculations.py

```diff
@@ -320,6 +320,16 @@ def test_mass_quantile(self):
         six.assertCountEqual(self, list(res.index), expected_index)
         self.assertAlmostEqual(res["TEST__index_mass_quantile__q_0.5"], 0.5, places=1)
 
+        # Test for parts of pandas series
```

Comment:
I just copied the example from above and added some values. It does not matter how much.